pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -240,6 +240,13 @@ def list_tasks(self, **kwargs) -> Dict:
         view = kwargs.get("view", "BASIC")
         projection = self._set_projection(view=view)
 
+        name_prefix = kwargs.get("name_prefix")
```

Comment:
For clarity, you could add a type hint for `name_prefix`, like so:

```python
name_prefix: str = ...
```

Reply:
Okay, will add it in my next commit

Reply:
> Looks great, very clean :) Just one minor suggestion.
> 
> @Ayush5120: Once @SohamRatnaparkhi has added the type hint, can you please test the feature and, if it works, approve the PR? Thanks!
Sure, I will do that